addons/website/views/snippets/s_cta_badge.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template name="CTA Badge" id="s_cta_badge">
+    <span class="s_cta_badge d-inline-block mb-3 py-2 px-3 border rounded-5 o_cc o_cc1 o_animable" data-name="CTA Badge"  style="border-width: 1px;">
```

Comment:
If you change the border width in the "CTA Badge" options to 2px and then change it back to 1px, a BS "border" class will be added here instead of using inline style. We always need to write the snippet XML in the same way the editor options do.
```suggestion
    <span class="s_cta_badge d-inline-block mb-3 py-2 px-3 border rounded-5 o_cc o_cc1 o_animable border" data-name="CTA Badge">
```

Reply:
In the same idea, for `Round Corners`, instead of using the class `rounded-5`, the editor add the `rounded` class and inline style (`border-radius: 32px !important;`).